cobc/typeck.c

```diff
@@ -5404,12 +5412,16 @@ expr_chk_cond (cb_tree expr_1, cb_tree expr_2)
 	return is_ptr_1 ^ is_ptr_2;
 }
 
+/* Reduce expressions of the left of 'token' that have higher priority */
 static int
-expr_reduce (int token)
+cb_expr_reduce (int token)
```

Comment:
we are not 100% clean on this yet, but as a rule of thumb: `cb_` are externalized functions in cobc, `cob_` externalized in libcob, otherwise these prefixes should not be usd

Reply:
Indeed, I renamed two functions because they were the 2 only ones that didn't have this prefix in a clique of functions that were actually _all_ static... So, I should have done the contrary :-)

Reply:
I would suggest to merge this PR and fix the prefixes later :-)

Reply:
I'm fine with fixing old prefixes later, but please revert the "bad" change here (or ensure later to not commit it) to not let the lines show up on git/svn blame for an unrelated change, then change back

Reply:
Ok, I have fixed it by changing the prefix from `cb_expr` to `build_expr` for all the functions in this clique.